docs/language/ql-handbook/about-the-ql-language.rst

```diff
@@ -1,17 +1,20 @@
-About QL
-========
+About the QL language
+######################
 
-This section is aimed at users with a background in general purpose programming as well as in databases. For a basic introduction and information on how to get started, see :doc:`Introduction to QL <introduction-to-ql>` and :doc:`Learning CodeQL <../index>`.
+QL is the powerful query language that underlies CodeQL, which is used to analyze code.
 
-QL is a declarative, object-oriented query language that is optimized to enable efficient analysis of hierarchical data structures, in particular, databases representing software artifacts.
-
-The queries and metrics used in LGTM are implemented using CodeQL, which uses QL to analyze code. This ensures that they can be extended or revised easily to keep up with changes in definitions of best coding practice. We continually improve existing queries as we work towards the ultimate goal of 100% precision.
+For more information about the important concepts and syntactic constructs of QL, see the individual reference topics such as :doc:`Predicates <predicates>` and :doc:`Types <types>`.
+The explanations and examples help you understand how the language works, and how to write more advanced QL code.
```

Comment:
I'm not sure whether this is the right place for these links. Possibly we could leave the TOC in the sidebar to advertise the first set of links? Perhaps we could link to the specifications from the "Properties of QL" section?

Reply:
That's a good point. I've moved the links down to the "properties" section and changed them so they're relevant to the code example.